scripts/management_review.py

```diff
@@ -20,16 +20,19 @@
 # Expected processes
 processes = [
     'Dx Sample registratie zuivering', 'Dx Hamilton uitvullen', 'Dx Hamilton zuiveren',
-    'Dx Zuiveren gDNA manueel', 'Dx manueel gezuiverd placement', 'Dx gDNA Normalisatie Caliper', 'Dx Fragmenteren & BBSS',
-    'Dx LibraryPrep Caliper KAPA', 'Dx Library Prep amplificatie & clean up KAPA', 'Dx Multiplexen library prep',
-    'Dx Enrichment DNA fragments', 'Dx Post Enrichment clean up', 'Dx Aliquot Post Enrichment (clean)',
-    'Dx Post Enrichment PCR & clean up', 'Dx Aliquot Post Enrichment PCR (clean)',
-    'Dx Library pool verdunnen', 'Dx Multiplexen library pool', 'Dx Multiplexen sequence pool',
-    'Dx Library pool denatureren en laden (NovaSeq)', 'Dx Automated NovaSeq Run (standaard)',
-    'Dx Library pool denatureren en laden (Nextseq)', 'Dx NextSeq Run',
+    'Dx Zuiveren gDNA manueel', 'Dx manueel gezuiverd placement', 'Dx gDNA Normalisatie Caliper', 'Dx Fragmenteren',
+    'Dx Library Prep & Target Enrichment Magnis', 'Dx Placement Enrichment Magnis',
+    'Dx 3nM verdunning Magnis', 'Dx Multiplexen Enrichment samples (3nM) Magnis', 'Dx Multiplexen Enrichment pools Magnis',
+    'Dx Multiplexen sequence pool', 'Dx Library pool denatureren en laden (NovaSeq)', 'AUTOMATED - NovaSeq Run (NovaSeq 6000)',
     'Dx Library pool denatureren en laden (NovaSeq) Dx QC controle Lab sequencen',
-    'Dx Library pool denatureren en laden (Nextseq) Dx QC controle Lab sequencen',
     'Dx NGS labwerk afronden', 'Dx Bioinformatica analyses', 'Dx NGS onderzoeken afronden',
+
+    # CREv2
+    'Dx Fragmenteren & BBSS', 'Dx LibraryPrep Caliper KAPA', 'Dx Library Prep amplificatie & clean up KAPA',
+    'Dx Multiplexen library prep', 'Dx Enrichment DNA fragments', 'Dx Post Enrichment clean up',
+    'Dx Aliquot Post Enrichment (clean)', 'Dx Post Enrichment PCR & clean up', 'Dx Aliquot Post Enrichment PCR (clean)',
+    'Dx Library pool verdunnen', 'Dx Multiplexen library pool', 'Dx Automated NovaSeq Run (standaard)',
+    'Dx Library pool denatureren en laden (NovaSeq) Dx QC controle Lab sequencen',
```

Comment:
I noticed that the process 'Dx Library pool denatureren en laden (NovaSeq) Dx QC controle Lab sequencen' is twice in processes = [...] (line 27 and line 35). I can't see if this means it (later on) is counted twice or not. Was this your intention?

Reply:
Nop, removed one :).